x-pack/plugins/lists/server/services/exception_lists/get_exception_list_summary.ts

```diff
@@ -84,7 +88,12 @@ export const getExceptionListSummary = async ({
     (acc, item: ByOsAggBucketType) => ({
       ...acc,
       [item.key]: item.doc_count,
-      total: acc.total + item.doc_count,
+      total:
+        // Do not add up the items by OS if host isolation exception
+        // As each host exception entry applies to all OSs
+        listId === ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_ID
```

Comment:
I'm not sure about this here. Exception items can have one or more OSs, even though for most of our types (trusted apps, event filters) they must be single OS. I think Endpoint Rule exceptions are also multi-OS entries, so host isolation is not the only one.

Because this API is "generic" against any exception list, I think the total should be the actual total number of items, not necessary be representative of the sum of items per-OS.

@dasansol92 , @FrankHassanabad , @yctercero : thoughts?

Reply:
done bdc0c97f1f7835f7560e6f725d7663800ac016f2